2-mandatory/1-weather-report.js

```diff
@@ -11,11 +11,14 @@
         - Hint: you can call the temperatureService function from your function
 */
 
+
 function getTemperatureReport(cities) {
-    // TODO
+    let arr = [];
+    for (let i = 0; i < cities.length; i++){
+        arr.push("The temperature in " + cities[i] + " is " + temperatureService(cities[i]) + " degrees");
+    } return arr;
```

Comment:
Minor point, but it would be good to put the return statement on a new line here for readability.